SSA/Projects/InstCombine/LLVM/Semantics.lean

```diff
@@ -4,23 +4,37 @@ import Mathlib.Tactic.Cases
 import Mathlib.Tactic.SplitIfs
 import Mathlib.Tactic.Tauto
 import SSA.Projects.InstCombine.LLVM.SimpSet
+import SSA.Core.Util
 
 
 namespace LLVM
 
+
+def IntN w := Option (BitVec w)
+
+def and??? {w: Nat} (x y : IntN w) : IntN w :=
```

Comment:
why not add this to the IntN namespace and call it just `and`? (i.e. `IntN.and`)

Reply:
This was just test code. It is now removed. The actual new methods are called `LLVM.and`.